src/application/topic_manager.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright 2025 Victor Drobizov
+
+#include <grrc/application/topic_manager.hpp>
+
+namespace grrc::application
+{
+
+bool TopicManager::Create(std::unique_ptr<application::abstract::ITopic> && topic)
+{
+  if (!topic) {
+    return false;
+  }
+
+  if (m_topics.end() == m_topics.find(topic->GetName())) {
```

Comment:
```suggestion
  if (!(m_topics.end() == m_topics.find(topic->GetName()))) {
```

в остальных менеджерах так же